src/StickyNote/StickyNote.tsx

```diff
@@ -1,16 +1,16 @@
 /* eslint-disable @typescript-eslint/no-explicit-any */
 
-import styles from './stickyNote.module.css';
+import style from "./stickyNote.module.css";
 
-import { useEffect, useState } from 'react';
-import { createSticky } from '../api_calls/create-sticky';
-import { getStickies } from '../api_calls/get-stickies';
-import { deleteSticky } from '../api_calls/delete-sticky';
-import { useNavigate } from 'react-router-dom';
+import { useEffect, useState } from "react";
+import { createSticky } from "../api_calls/create-sticky";
+import { getStickies } from "../api_calls/get-stickies";
+import { deleteSticky } from "../api_calls/delete-sticky";
+import { useNavigate } from "react-router-dom";
 
 const StickyNote = () => {
-  const [title, setTitle] = useState<string>('');
-  const [note, setNote] = useState<string>('');
+  const [title, setTitle] = useState<string>("");
+  const [note, setNote] = useState<string>("");
   const [stickies, setStickies] = useState<any>([]);
 
   const navigate = useNavigate();
```

Comment:
for routing to pages,, please use `() => navigate('pathName')`

Its the proper way to use react router, it caused a deployment check because this Navigate variable is unused. 

please update :)